auth/view.py

```diff
@@ -0,0 +1,47 @@
+from abc import ABC, abstractmethod
+
+from fastapi import Request
+from fastapi.templating import Jinja2Templates
+from jinja2 import Environment
+import jinja2
+
+from auth.controller import AuthController
+from auth.service import AuthService
+
+from auth.middleware import login_required
+
+class UserView(ABC):
+    @abstractmethod
+    def view_users(self, request: Request):
+        pass
+      
+class UserViewV1(UserView):    
+    def __init__(self, controller: AuthController, service: AuthService, admin_emails:list[str]) -> None:
+        super().__init__()
+        self.templates = Jinja2Templates(
+            env=Environment(
+                loader=jinja2.FileSystemLoader(['auth/templates', "bot/templates"]),
```

Comment:
Do we need to load the template files from another module, `bot`?

Reply:
@TakeMeGH

Reply:
The navbar is currently in the bot module. Maybe we can adjust the HTML location for the navbar later, but for now, I'm importing it from the bot module.

Reply:
Ah.. I see. Yeah, a refactoring task for another day it seems. I'll put it on our Linear backlog for now

Reply:
https://linear.app/sapujagad-id/issue/SAP-55/refactor-navbar-from-bot-module